client/account/AccountProfileForm.js

```diff
@@ -0,0 +1,214 @@
+import React, { useCallback, useMemo, useEffect, useState } from 'react';
+import { Field, FieldGroup, TextInput, TextAreaInput, Box, Icon, AnimatedVisibility, PasswordInput, Button, Grid } from '@rocket.chat/fuselage';
+import { useDebouncedCallback } from '@rocket.chat/fuselage-hooks';
+
+import { useTranslation } from '../contexts/TranslationContext';
+import { isEmail } from '../../app/utils/lib/isEmail.js';
```

Comment:
Can this import break Storybook?

Reply:
I don't think so. It's a static regex text...